pkg/cmd/drtprod/scripts/tpcc_init.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+# Copyright 2024 The Cockroach Authors.
+#
+# Use of this software is governed by the CockroachDB Software License
+# included in the /LICENSE file.
+
+# This script sets up the tpcc import workload script in the workload node and starts the same in nohup
+# NOTE - This uses CLUSTER and WORKLOAD_CLUSTER environment variable, if not set the script fails
+
```

Comment:
It would be good to add 
```
export ROACHPROD_GCE_DEFAULT_PROJECT=cockroach-drt
export ROACHPROD_DNS="drt.crdb.io"
./roachprod sync
sleep 20
```

Before we start off the script. In some of my testing I saw some `cluster not found` errors

Reply:
I meant to add this in the generated script, i.e., `tpcc_init.sh`